.rhcicd/floorist/floorplan.yaml

```diff
@@ -216,11 +216,68 @@ objects:
           INNER JOIN
             bundles
               ON bundles.id = applications.bundle_id
+- apiVersion: metrics.console.redhat.com/v1alpha1
+  kind: FloorPlan
+  metadata:
+    name: notifications-hms
+  spec:
+    database:
+      secretName: ${FLOORIST_DB_SECRET_NAME}
+    objectStore:
+      secretName: ${FLOORIST_HMS_BUCKET_SECRET_NAME}
+    logLevel: ${FLOORIST_LOGLEVEL}
+    suspend: ${{FLOORIST_SUSPEND}}
+    disabled: ${{FLOORIST_DISABLED}}
```

Comment:
We don't have the `FLOORIST_DISABLED` parameter in our template. Feel free to either remove it from this FloorPlan or to add it in our parameters below.

Reply:
Thank you. Removed!